ovirtlago/cmd.py

```diff
@@ -223,22 +231,30 @@ def do_ovirt_status(prefix, **kwargs):
     dest='with_vms',
     action='store_true',
 )
+@cli_plugin_add_argument(
+    '--vms-timeout',
+    help=('Time to wait until the Engine reports the VMs are up.'),
+    dest='vms_timeout',
+    default=8 * 60,
+    type=int,
+    action='store',
+)
 @in_ovirt_prefix
 @with_logging
-def do_ovirt_start(prefix, with_vms, **kwargs):
+def do_ovirt_start(prefix, with_vms, vms_timeout, **kwargs):
     with LogTask('Starting oVirt environment'):
         prefix.start()
         with LogTask('Waiting for ovirt-engine status'):
-            prefix.virt_env.assert_engine_alive()
+            prefix.virt_env.assert_engine_alive(timeout=2 * 60)
```

Comment:
Could be - this would mean it takes libvirt to boot the VM and until the engine service is running more than 2 minutes, which seems a lot. I never had it timed out on that stage, though I never had it timed out on the 'vms' start either.  I actually think that the timeout for the VMs now is huge - 8 minutes. But practically I can't think of any other 'hot fix' for that..

You experienced it timing out on this engine service stage?